content/de/cloud_computing.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Cloud Computing
+status: Feedback erwünscht
```

Comment:
@jihoon-seo oops, totally forgot about that ^^. Will add a PR

Reply:
According to this line:
https://github.com/cncf/glossary/blob/f71986bf755227667c9ba127d20709e4c923bcc1/layouts/partials/sidebar-tree.html#L43

it seems that as you said, only pages with `status` either `Completed` or `completed` are exposed, while those with other strings aren't.

So IMO
- both `Feedback erwünscht` and `Feedback appreciated` are good (I mean, make no difference for exposure),
- but it will be easier for non-German speakers (e.g. @jihoon-seo) to notice the status if the status is written in English. 😊

Reply:
Fair enough I guess we need to make some changes then :)